experimental/babel-preset-env/src/debug.js

```diff
@@ -9,7 +9,7 @@ const wordEnds = size => {
 export const logMessage = (message, context) => {
   const pre = context ? `[${context}] ` : "";
   const logStr = `  ${pre}${message}`;
-  console.log(logStr);
+  console.error(logStr);
```

Comment:
Personally it seems like `console.error` implies an error condition to me. Should these be `console.warn` perhaps?

Reply:
I agree, I would even prefer a layer between that, like just `output(msg: string)` which calls the appropriate console method.

Reply:
Hmm, is `output` a good name? The js is also output…

Reply:
Righ, maybe `printDebug(msg)`?